Clean-up YANG groupings and components identifiers (#65)
ietf-ivy-wg/network-inventory-yang: draft-ietf-ivy-network-inventory-yang.md

### Component-Specific Info Design

> According to the management requirements from operators, some important attributes are not defined in {{!RFC8348}}. These attributes could be component-specific and are not suitable to define under the component list node. So, component-specific attributes can be defined by augmenting the component-specific info containers for the attributes applicable to HW e.g. boards/slot components only. Other component-specific attributes, such as SW-specific-info, may be defined in companion augmentation data models, such as

Reviewer: %s/not suitable to define/not suitable to be defined

%s/So, ... components only/ Instead, they can be defined by augmenting the component-specific info container for the attributes applicable to HW...only.

Author: ```suggestion
According to the management requirements from operators, some important attributes are not defined in {{!RFC8348}}. These attributes could be component-specific and are not suitable to be defined under the component list node. Instead, they can be defined by augmenting the component-specific info container for the attributes applicable to HW e.g. boards/slot components only. Other component-specific attributes, such as SW-specific-info, may be defined in companion augmentation data models, such as
```
